### ISSUE TITLE
Add learning task to predict LOS based on coding

### ISSUE THREAD

**Issue by tschimbr** [eonum/medcodelearn#28]
This would be a regression task.

**Comment by tschimbr** [comment]
A benchmark could be the average length of the corresponding DRG

**Comment by tschimbr** [comment]
See los_prediction.py

**Comment by tschimbr** [comment]
Results: 

Global average LOS: 6.33675
Total test MAPE global average baseline: 1.71994
Total test MSE global average baseline: 76.6839
Total test MAE global average baseline: 4.98716
Total test MAPE DRG baseline: 0.875988365962
Total test MSE DRG baseline: 42.3665166512
Total test MAE DRG baseline: 3.33587163728
Training data dimensionality: 227952 | 20
Train LSTM Neural Net with Embedding..
Train on 129817 samples, validate on 22910 samples
Epoch 1/50
133s - loss: 2.7602 - mean_absolute_percentage_error: 56.3484 - mean_squared_error: 38.5304 - val_loss: 2.5947 - val_mean_absolute_percentage_error: 51.8078 - val_mean_squared_error: 32.0187
Epoch 2/50
144s - loss: 2.4193 - mean_absolute_percentage_error: 48.3093 - mean_squared_error: 30.4986 - val_loss: 2.5084 - val_mean_absolute_percentage_error: 48.2677 - val_mean_squared_error: 29.7028
Epoch 3/50
145s - loss: 2.2675 - mean_absolute_percentage_error: 45.1468 - mean_squared_error: 26.8567 - val_loss: 2.5030 - val_mean_absolute_percentage_error: 46.5308 - val_mean_squared_error: 28.8721
Epoch 4/50
143s - loss: 2.1514 - mean_absolute_percentage_error: 43.2441 - mean_squared_error: 23.8941 - val_loss: 2.4868 - val_mean_absolute_percentage_error: 50.5930 - val_mean_squared_error: 26.3476
Epoch 5/50
144s - loss: 2.0422 - mean_absolute_percentage_error: 41.3010 - mean_squared_error: 21.5533 - val_loss: 2.4749 - val_mean_absolute_percentage_error: 46.5228 - val_mean_squared_error: 27.9491
Epoch 6/50
144s - loss: 1.9477 - mean_absolute_percentage_error: 39.6629 - mean_squared_error: 19.2869 - val_loss: 2.5125 - val_mean_absolute_percentage_error: 48.5646 - val_mean_squared_error: 27.5131
Epoch 7/50
144s - loss: 1.8646 - mean_absolute_percentage_error: 38.2895 - mean_squared_error: 17.8183 - val_loss: 2.4977 - val_mean_absolute_percentage_error: 48.5884 - val_mean_squared_error: 27.5652
Epoch 8/50
144s - loss: 1.7891 - mean_absolute_percentage_error: 37.0385 - mean_squared_error: 16.2144 - val_loss: 2.5039 - val_mean_absolute_percentage_error: 49.7746 - val_mean_squared_error: 27.6760
Epoch 9/50
144s - loss: 1.7273 - mean_absolute_percentage_error: 35.9496 - mean_squared_error: 15.1459 - val_loss: 2.5043 - val_mean_absolute_percentage_error: 48.9118 - val_mean_squared_error: 27.8385
Epoch 10/50
144s - loss: 1.6698 - mean_absolute_percentage_error: 34.9717 - mean_squared_error: 13.8940 - val_loss: 2.5209 - val_mean_absolute_percentage_error: 47.9585 - val_mean_squared_error: 27.4961
Epoch 11/50
144s - loss: 1.6114 - mean_absolute_percentage_error: 33.8195 - mean_squared_error: 12.9108 - val_loss: 2.5519 - val_mean_absolute_percentage_error: 49.9782 - val_mean_squared_error: 28.1428
Epoch 12/50
144s - loss: 1.5648 - mean_absolute_percentage_error: 33.0037 - mean_squared_error: 12.2485 - val_loss: 2.5873 - val_mean_absolute_percentage_error: 52.2681 - val_mean_squared_error: 27.7438
Epoch 13/50
144s - loss: 1.5138 - mean_absolute_percentage_error: 32.2407 - mean_squared_error: 11.1705 - val_loss: 2.5493 - val_mean_absolute_percentage_error: 49.7420 - val_mean_squared_error: 27.7330
Epoch 14/50
144s - loss: 1.4680 - mean_absolute_percentage_error: 31.4446 - mean_squared_error: 10.5042 - val_loss: 2.6257 - val_mean_absolute_percentage_error: 51.4342 - val_mean_squared_error: 28.5332
Epoch 15/50
145s - loss: 1.4326 - mean_absolute_percentage_error: 30.7326 - mean_squared_error: 9.9355 - val_loss: 2.5855 - val_mean_absolute_percentage_error: 49.2873 - val_mean_squared_error: 28.1406
Epoch 16/50
145s - loss: 1.3913 - mean_absolute_percentage_error: 29.9902 - mean_squared_error: 9.3498 - val_loss: 2.6151 - val_mean_absolute_percentage_error: 49.7188 - val_mean_squared_error: 28.6507
Prediction using LSTM..
Test MAE: 2.59104322489
Test MAPE: 49.9006595713
Total test MAPE: 0.499007
Total test MSE: 28.9298
Total test MAE: 2.59104
